File-Formats/DocIO/Working-with-Charts.md

```diff
@@ -2382,6 +2382,270 @@ You can download a complete working sample from [GitHub](https://github.com/Sync
 N> 1. To convert chart in Word document as image, it is need to refer chart conversion related [assemblies](https://help.syncfusion.com/file-formats/docio/assemblies-required#converting-charts) or [NuGet packages](https://help.syncfusion.com/file-formats/docio/nuget-packages-required#converting-charts).
 N> 2. The ChartToImageConverter is supported from .NET Framework 4.0 onwards.
 
+## Word 2016 Charts
+
+Essential DocIO supports creating and manipulating new and modern chart types such as waterfall, which are introduced in Microsoft Word 2016.
+
+### Waterfall
+
+[Waterfall](https://support.office.com/en-us/article/Create-a-waterfall-chart-in-Office-2016-8de1ece4-ff21-4d37-acd7-546f5527f185#) chart helps understand the finances of business owners by viewing profit and loss statements. You can quickly illustrate the line items in your financial data and get a clear picture of how each item is impacting your bottom line using a Waterfall chart. Refer to the following code to create a Waterfall chart.
```

Comment:
Refer to the following code to create a Waterfall chart. 

Please modify this line

Reply:
Modified